lib/simple_token_authentication.rb

```diff
@@ -46,7 +46,7 @@ def self.load_available_adapters adapters_short_names
   end
 
   def self.adapter_dependency_fulfilled? adapter_short_name
-    qualified_const_defined?(SimpleTokenAuthentication.adapters_dependencies[adapter_short_name])
+    const_defined?(SimpleTokenAuthentication.adapters_dependencies[adapter_short_name])
```

Comment:
I would not include this in the Devise 4 support, because it is part of supporting Rails 5 and it has little to do with supporting Devise 4 itself.